install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -175,6 +175,16 @@ openshift_node_open_ports:
   port: 53/tcp
 - service: dns udp
   port: 53/udp
+
+# To set the pod network CIDR range, uncomment the following property and set its value.
+# The value must be smaller than the CIDR value set in the inventory file as
```

Comment:
It might make more sense if this line was called out as a note, though not necessarily with full admonition styling cuz that might look odd. The way it reads now ("uncomment the following") caused me to have to think a minute about which line to uncomment. And maybe add the property in backticks?

To set the pod network CIDR range, uncomment the following property and set its value.
Note that the value must be smaller than the CIDR value set in the inventory file as
openshift_openstack_kuryr_pod_subnet_cidr. By default, this value is /24.

`openshift_openstack_kuryr_pod_subnet_prefixlen: 24`

Then again, adding "note that" might not help clarify anything. ¯\_(ツ)_/¯

Reply:
@bobfuru I agree that it's not necessarily the most readable treatment. It's a copy of text in a sample file in openshift-ansible that (ostensibly) stands on its own, which makes it a hairier call, IMO. 

On another pass, I think that "set its value" should end in a colon w/the property on the next line. "The value must be smaller than the CIDR value..." and other information would come after the property. Thoughts?

Reply:
Yes, that makes more sense. 👍  Simple yet brilliant! I was going to suggest a colon before but it didn't make sense without the property following it. This way, it does.